adal/authority.py

```diff
@@ -118,7 +127,7 @@ def _perform_dynamic_instance_discovery(self):
         operation = "Instance Discovery"
         self._log.debug("Attempting instance discover at: %(discovery_endpoint)s",
                         {"discovery_endpoint": discovery_endpoint.geturl()})
-
+                        	

```

Comment:
Nitpicking: would you mind remove the unnecessary space(s) introduced in this PR? Currently they are displayed as changes in this github PR page and then become a distraction during Code Review, twice: one time in this PR review, and another time in future when another person modify this file again and his/her editor is configured to trim trailing spaces automatically.

Reply:
Removed unnecessary space